package.json

```diff
@@ -201,6 +205,7 @@
     "fine-uploader": "^5.11.8",
     "font-awesome": "^4.6.3",
     "foundation-sites": "^5.5.3",
+    "graphql": "^14.0.2",
```

Comment:
There's an example of consuming Drupal data via a traditional JSON API at https://github.com/department-of-veterans-affairs/va.gov-cms/blob/41c149a11858124256d2fbcfbc4c7d9798654db6/metalsmith_app/custom_plugins/metalsmith-getbasicpage/index.js. I went with the GraphQL option though because of the amount of content mapping that seems to be required with that approach.